### DISCUSSION TITLE
It doesn’t follow me back

### DISCUSSION THREAD

**Discussion by mcepl** [snarfed/bridgy-fed#1562] [Q&A] [answered]
> To bridge your fediverse account into Bluesky and interact with people there, search for and follow @bsky.brid.gy@bsky.brid.gy. That account will then follow you back. Accept its follow to make sure your fediverse posts get sent the bridge and make it into Bluesky.

No, it doesn’t. I tried a couple of times to follow it, but it never made any contact with me (my account is https://en.osm.town/@mcepl).

**Comment by snarfed** [comment, marked as answer]
Sorry for the trouble! Have you looked at https://fed.brid.gy/docs#troubleshooting ?

**Reply by Tamschi** [reply]
More precisely, the account is too recent. It was created 3 days ago, but at least 14 are required (to hopefully prevent spam waves).
Just unfollow and refollow once your account is old enough and it should work.

**Reply by mcepl** [reply]
I see. Right, I switched from https://floss.social to be able to use Bridgy Fed. So, this is my punishment :crying_cat_face: 

See ya in ten days!

**Reply by mcepl** [reply]
So, yes, I have refollowed now, it seems to work, I have confirmed the bot following me, I have started following one account (@anneapplebaum.bsky.social@bsky.brid.gy), but when I tried to add two more (@timothysnyder.bsky.social@bsky.brid.gy and @trent-j-telenko.bsky.social@bsky.brid.gy), https://en.osm.town/ is not able to find them. Is there still some rate limit even on adding new accounts?

**Reply by snarfed** [reply]
Looks like those accounts aren't bridged yet. Feel free to ask them to! https://fed.brid.gy/docs#dm-request

**Reply by Tamschi** [reply]
Neither has open Chat, so that won't work.

@mcepl You'll have to use another means to ask them, we don't have the ability to send them a message in a non-spammy way.
See also #1471 for discussion of what's needed to make Bluesky users available to follow via ActivityPub by default.
